### PR TITLE
Target size customization

### PR DESCRIPTION
**Pull request by udy18rus**

### CODE REVIEW

File: taptargetview/src/main/java/com/getkeepsafe/taptargetview/TapTarget.java [hunk 1]
```diff
@@ -371,6 +372,13 @@ public TapTarget id(int id) {
         return this;
     }
 
+    /** Specify the target size. **/
```
Comment (xiphirx) [inline taptargetview/src/main/java/com/getkeepsafe/taptargetview/TapTarget.java]: Please add `in dp` to this javadoc.

File: taptargetview/src/main/java/com/getkeepsafe/taptargetview/TapTarget.java [hunk 2]
```diff
@@ -371,6 +372,13 @@ public TapTarget id(int id) {
...
+    public TapTarget targetSize(int targetSize) {
```
Comment (xiphirx) [inline taptargetview/src/main/java/com/getkeepsafe/taptargetview/TapTarget.java]: Lets rename this to `targetRadius`

File: taptargetview/src/main/java/com/getkeepsafe/taptargetview/TapTarget.java [hunk 3]
```diff
@@ -425,6 +433,10 @@ int descriptionTextSizePx(Context context) {
         return dimenOrSize(context, descriptionTextSize, descriptionTextDimen);
     }
 
+    int targetSize() {
+        return targetSize;
```
Comment (xiphirx) [inline taptargetview/src/main/java/com/getkeepsafe/taptargetview/TapTarget.java]: This function doesn't need to exist.

File: taptargetview/src/main/java/com/getkeepsafe/taptargetview/TapTargetView.java [hunk 4]
```diff
@@ -354,7 +354,7 @@ public TapTargetView(Context context,
 
         TARGET_PADDING = UiUtil.dp(context, 20);
         CIRCLE_PADDING = UiUtil.dp(context, 40);
-        TARGET_RADIUS = UiUtil.dp(context, 44);
+        TARGET_RADIUS = UiUtil.dp(context, target.targetSize());
```
Comment (xiphirx) [inline taptargetview/src/main/java/com/getkeepsafe/taptargetview/TapTargetView.java]: You could just do `target.targetSize` or `target.targetRadius` after the renaming.

**Comment by udy18rus** [comment]
All ready.

**Comment by xiphirx** [comment]
Thanks!

**Pull request merged**